simpledotorg/simple-server
# Add "Achievements" card to Progress tab home page

**Story card:** [sc-7577](https://app.shortcut.com/simpledotorg/story/7577/new-achievements-card)

## Because

We're connecting updating "Achievements" section UI to show a list of the five most recent badges for follow up patients and registrations.

## This addresses
**Before**
![image](https://user-images.githubusercontent.com/16785131/158464348-0eb5e81a-db76-43de-b7ec-bb53d468523e.png)

**After**
![image](https://user-images.githubusercontent.com/16785131/158463805-5ee7b655-f654-4d18-8b4e-cfff13d7de6d.png)
* Added new "Achievements" card
* Deleted the old achievements and footer sections
* Reduced the font size for the gender/diagnosis `select` in the "Registrations" and "Follow-ups" cards [Praveen reported](https://simpledotorg.slack.com/archives/C013HS4N9U3/p1647318517496769) on 15-Mar-2022
* Refactored the `ProgressTabHelper` module
* Deleted the `FacilityProgressAchievementService`

## Test instructions

1. Open your local and go to the Report page of any facility
2. Click the "Progress reports" link
3. Make note of the "All-time" registration and follow-up values displayed in the respective cards
4. Scroll to the end of the page and check the "Achievements" card
5. The follow-up and registration values should match the values displayed in the cards above
6. There should be 5 badges for follow-ups and 5 badges for registrations
7. If there are `0` follow-ups or registrations, all 5 badges should be outlined in gray
8. The last badge should always be outlined in gray
9. If the follow-ups/registrations value is equal to the 4th badge (second-to-last badge) the badge should be colored in (For example, if there are 250 follow-ups, the 4th badge should read "250" and it should be colored in).
